syndicate/connection/eventbridge_schedule_connection.py

```diff
@@ -0,0 +1,49 @@
+from boto3 import client
+from botocore.exceptions import ClientError
+
+from syndicate.commons.log_helper import get_logger
+from syndicate.connection.helper import apply_methods_decorator, retry
+
+_LOG = get_logger('syndicate.connection.eventbridge_schedule_connection')
+
+
+@apply_methods_decorator(retry)
+class EventBridgeScheduleConnection(object):
```

Comment:
Service is called AWS EventBridge Schedule<ins>r</ins>
Please rename the file and respective classes

Reply:
done